helm_deploy/values-dev.yaml

```diff
@@ -25,6 +25,7 @@ generic-service:
     API_BASE_URL_ALERTS: https://alerts-api-dev.hmpps.service.justice.gov.uk
     API_BASE_URL_COURT_SENTENCING: https://remand-and-sentencing-api-dev.hmpps.service.justice.gov.uk
     API_BASE_URL_PRISON_PERSON: https://prison-person-api-dev.prison.service.justice.gov.uk
+    FEATURE_EVENT_OFFENDER_PHYSICAL_ATTRIBUTES-CHANGED: false
```

Comment:
Turning this sync off temporarily to allow prison person API to make a breaking change